README.md

```diff
@@ -37,3 +43,12 @@ await.allSettled expr
 await.any expr
 // eq: await Promise.any(expr)
 ```
+
+## Alternative solution
+
+Introduce a new global `Async` (or `Await`), then let
+
+- `Async.all` (equals to `Promise.all`)
+- `Async.race` (equals to `Promise.race`)
+- `Async.allSettled` (equals to `Promise.allSettled`)
+- `Async.any` (equals to `Promise.any`)
```

Comment:
I’m very confused by this - adding a new global isn’t going to address the confusion caused by the ability to use syntax for await but not for the combinators.

Reply:
oh yeah, I found this way we need `await Async.all(...)` which is, ... a bit strange. let me remove this part.